src/petclaw/tests/test_io.py

```diff
@@ -21,4 +21,7 @@ def test_data_dir(self):
         return os.path.join(self.this_dir, '../../pyclaw/tests/test_data')
 
     def test_io_from_binary(self):
-        return
\ No newline at end of file
+        return
+
+    def test_io_to_vtk(self):
```

Comment:
Actually VTK output is already in petclaw (just grep for vtk in the petclaw directory).

Reply:
Found it! Thanks for pointing this out.

Reply:
@ketch I have a clarification question on this point. 

The vtk test that I've written relies on reading in a binary file and then verifying that the output written is the same as a sample vtk file. I wrote the test this way because there was a simple binary file in the test data folder already, and because this allows me to test the vtk output without a vtk writer (would be nice, but not a priority at the moment). 

When I remove the redefinition I get the following message:
```
ERROR: test_io.PetClawIOTest.test_io_to_vtk
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/barnhark/opt/anaconda3/envs/clawpack/lib/python3.8/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/Users/barnhark/projects/clawpack/pyclaw/src/pyclaw/tests/test_io.py", line 76, in test_io_to_vtk
    ref_sol.read(0,path=regression_dir,file_format="binary")
  File "/Users/barnhark/projects/clawpack/pyclaw/src/pyclaw/solution.py", line 343, in read
    read_func = self.get_read_func(file_format)
  File "/Users/barnhark/projects/clawpack/pyclaw/src/petclaw/solution.py", line 17, in get_read_func
    raise ValueError("File format %s not supported." % file_format)
ValueError: File format binary not supported.
```
Because my test relies on reading binary, and this error indicated to me that petclaw can't read binary, I redefined the function, just as `test_io_from_binary` was redefined. As such, I think that it makes sense to retain this redefinition. 

If you have other recommendations for how to handle this, I'd be happy to hear them.